js/count-letter-in-string.js

```diff
@@ -0,0 +1,15 @@
+// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript
+
+function count(string) {
+  const lettersCount = {};
+  for (let i = 0; i < string.length; i++) {
+    if (lettersCount[string[i]] == undefined) {
```

Comment:
What's the reason for `==`?

If that's by mistake, please configure eslint to enforce using `===`

Reply:
Are you going to configure eslint :smile: ?